.circleci/config.yml

```diff
@@ -11,34 +11,34 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v1-gem-cache-{{ checksum "Gemfile.lock" }}
-            - v1-gem-cache
+            - npm-cache-{{ checksum "package-lock.json" }}
+      - restore_cache:
+          keys:
+            - gem-cache-{{ checksum "Gemfile.lock" }}
+            - gem-cache
       - run:
           name: Install ruby dependencies
           command: |
             gem update --system
             gem install bundler
             bundle install --jobs=4 --retry=3 --path vendor/bundle
+      - run:
+          name: Install scss_lint
+          command: gem install scss_lint
       - save_cache:
-          key: v1-gem-cache-{{ checksum "Gemfile.lock" }}
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
           paths:
             - vendor/bundle
-      - restore_cache:
-          keys:
-            - v2-npm-cache-{{ .Branch }}-{{ .Revision }}
       - run:
           name: Install node dependencies
           command: npm install
+      - run:
```

Comment:
Adds a build step, building the USWDS `dist` directory if necessary. This allows those build files to be part of the node installation cache.